src/scripts/main.js

```diff
@@ -1,14 +1,32 @@
 'use strict';
 
 const pushNotification = (posTop, posRight, title, description, type) => {
-  // write code here
+  const notification = document.createElement('div');
+  const h2 = document.createElement('h2');
+  const p = document.createElement('p');
+
+  notification.className = `notification ${type}`;
+  notification.style.top = `${posTop}px`;
+  notification.style.right = `${posRight}px`;
+  notification.style.boxSizing = 'inherit';
+  h2.className = 'title';
+  h2.textContent = title;
+  p.innerHTML = description.replace(/\n/g, '<br>');
+
+  document.body.appendChild(notification);
+  notification.appendChild(h2);
+  notification.appendChild(p);
+
+  setTimeout(() => {
+    notification.remove();
```

Comment:
Consider using styles to remove an element to pass tests

Reply:
Do you mean like this?
setTimeout(() => {
    notification.style.display = 'none';
  }, 2000);